### ISSUE TITLE
For "Select subscriptions" combo box, programmatic name is given as input: A11y_Azure Machine Learning Extension for VS Code_Appling filter on subscription_Label in name

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode-tools-for-ai#2488]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev2; #BM_Azure Machine Learning Extension for VS Code_Win32_Jan2024; #Azure Machine Learning Extension for VS Code; #WCAG2.5.3; #FTP; #Label In name; #A11yWCAG2.1; #External; #External:VSCode; #Firstpart:Bug245219; #Regressed:04-17-25; #Closed; #A11yFixed;

## Environment Details:
Application: Azure Machine Learning Extension for VS Code
Version: 1.2.0
Visual studio code version: 1.96.3
OS: Windows 11 Enterprise 24H2
Build: 26100.2605

## Repro Steps

1. Launch the VS Code.
2. Install the Azure Machine Learning extension and sign in with appropriate Credentials.
3. Tab until the left toolbar (list of icons). This toolbar has a file icon, search icon, settings, etc.
4. Use the up/down arrow keys to navigate to the "Azure" icon and press enter
5. The Azure panel will now be active next to the toolbar.
6. Tab until the Machine Learning dropdown.
7. Tab till any subscription for example AML V1 Personal subscription.
8. Tab until the "Filter" button and press Enter. It will be a button with a funnel icon.
9. TAB till "Select subscriptions" combo box and inspect it.
10. Observer that for "Select subscriptions" combo box, programmatic name is given as input.

## Actual Experience
For "Select subscriptions" combo box, programmatic name is given as input.

### Similar issue observed with following scenarios.

**Scenario1**: Machine learning _Creating a job

1. TAB till create job button and press ENTER key
2. TAB till Combo box and inspect it

**Scenario2**: Machine learning _Setting Default Workspace

1. Open the Command Palette via the keyboard command Ctrl + Shift + P 
2. Type "Set default workspace" and select the option from the dropdown list called "Azure ML: Set Default Workspace") and press enter
3. A list should appear that says "Select a subscription"
4. Use the up/down arrows to navigate the list until "AML V1 Personal 2" is selected, and press enter
5. A list should appear that says "Select a workspace"
6. TAB till Select a workspace combo box and inspect it.

## Expected Experience
For "Select subscriptions" combo box, programmatic name should be same as visual label name.

## User Impact:
Screen reader user and speech input user will face difficulty, screen reader user will not understand the purpose of combo box because screen reader will announce as input combo box and speech input use will face difficulty to interact because programmatic and visual label is not same.

## Attachment
![Image](https://github.com/user-attachments/assets/96d4fc19-edbf-470c-a7aa-c82be6a08db6)

Similar issue 1
![Image](https://github.com/user-attachments/assets/0968e96a-7398-436c-aec0-9ab20be0c8a7)

Similar issue 2

![Image](https://github.com/user-attachments/assets/b55110f0-4c6a-496b-97e4-bd6d0e356c79)

**Comment by JerryYangKai** [comment]
Hello @kapilvaishna
The Azure ML extension is a VS Code extension based on VS Code API, after checking the official doc there's no property to control the name 'input', so it is by design. Refer to this link https://vscode-api.js.org/interfaces/vscode.InputBox.html

I also compare Azure ML extension with other Microsoft official VS Code extension (the Azure Resources extension), the experience is the same.
This is ML extension select Azure subs

![Image](https://github.com/user-attachments/assets/21d79d3c-61b3-4cd8-83d6-24d832441f54)

This is Azure Resources extension select subs 

![Image](https://github.com/user-attachments/assets/0ff52aa0-8dfc-4e7e-9b9c-a602216509e7)

**Comment by kapilvaishna** [comment]
Issue is not repro in below environment so closing the bug

Visual studio code insider
Version 1.100.0


![Image](https://github.com/user-attachments/assets/00f314c2-2654-4826-918a-27fafecadfe0)


![Image](https://github.com/user-attachments/assets/87153b7f-cea4-4b08-bb35-a7951d8b82fd)


![Image](https://github.com/user-attachments/assets/959559f5-c87e-47db-8bf1-bc02e24121ca)
